File: GTTS/INL.TrainingService.Models/IGetTrainingEventParticipantAttachment_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public interface IGetTrainingEventParticipantAttachment_Item
    {
        string ParticipantType { get; set; }
        long TrainingEventParticipantAttachmentID { get; set; }
        long TrainingEventID { get; set; }
        long PersonID { get; set; }
        long FileID { get; set; }
        int FileVersion { get; set; }
        int TrainingEventParticipantAttachmentTypeID { get; set; }
        string TrainingEventParticipantAttachmentType { get; set; }
        string FileName { get; set; }
        string FileLocation { get; set; }
        byte[] FileHash { get; set; }
        int FileSize { get; set; }
        byte[] FileContent { get; set; }
        string ThumbnailPath { get; set; }
        string Description { get; set; }
        bool? IsDeleted { get; set; }
        int ModifiedByAppUserID { get; set; }
        DateTime ModifiedDate { get; set; }
        string FileJSON { get; set; }
        string ModifiedByUserJSON { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/iget-training-event-participant-attachment_item.ts




export interface IGetTrainingEventParticipantAttachment_Item {
  
	ParticipantType: string;
	TrainingEventParticipantAttachmentID: number;
	TrainingEventID: number;
	PersonID: number;
	FileID: number;
	FileVersion: number;
	TrainingEventParticipantAttachmentTypeID: number;
	TrainingEventParticipantAttachmentType: string;
	FileName: string;
	FileLocation: string;
	FileHash: number[];
	FileSize: number;
	FileContent: number[];
	ThumbnailPath: string;
	Description: string;
	IsDeleted?: boolean;
	ModifiedByAppUserID: number;
	ModifiedDate: Date;
	FileJSON: string;
	ModifiedByUserJSON: string;
}





